### ISSUE TITLE
Detect licenses during init

### ISSUE THREAD

**Issue by loleg** [datopian/data-cli#239]
In the `init` process described in #63 there could be a detection of a `LICENSE` or `LICENSE.md` file in the current folder, leading to the addition to `datapackage.json` of the appropriate license fields from the specification. A range of [open data licenses](https://opendatacommons.org/licenses/) could be supported, perhaps linked to [opendefinition.org](http://licenses.opendefinition.org/licenses/ODC-BY-1.0.json) to stay up to date.

See also: [detectable licenses on GitHub.com](https://help.github.com/articles/adding-a-license-to-a-repository/)

**Comment by anuveyatsu** [comment]
@loleg you can add ODC licenses during the `init` command now (available from v0.9.0). By default, "ODC-PDDL" license is added but you can run the command in the interactive mode using `-i` option and select another license. Do you think it addresses this issue?

<img width="240" alt="screen shot 2018-04-16 at 23 13 36" src="https://user-images.githubusercontent.com/17809581/38824675-d4898ca0-41cb-11e8-8a0f-86468a5fcc1a.png">

**Comment by loleg** [comment]
@anuveyatsu I don't think this completely responds to the suggestion, since I was thinking the license could be automatically detected. But for most intents and purposes, I think this will do the job. 

Is there a confirmation of the default license being added?
